internal/pkg/utils/testproject/snapshot.go

```diff
@@ -188,7 +188,7 @@ func (p *Project) NewSnapshot() (*fixtures.ProjectSnapshot, error) {
 
 	// Join buckets with tables
 	for _, t := range tables {
-		b := bucketsMap[t.Bucket.ID]
+		b := bucketsMap[t.ID.BucketID]
```

Comment:
V `t.Bucket` je `nil`. Potřebuju jen bucket id, tak nezkoumám proč. 🙂

Reply:
To by bolo dobre asi preskumat :slightly_smiling_face:  , aj dole, kde je prazdne `name, stage`, ..... to by malo byt vyplnene.

Nebude problem, ze sa to nacita bez nejakeho detailu?

Reply:
Rovnako aj `columns: nil`.

Reply:
omg, těch columns jsem si nějak nevšim. 😅 Tak to je jasný, prověřím.